spec/src/main/asciidoc/related-documents.adoc

```diff
@@ -5,22 +5,11 @@
 [bibliography]
 == Related Documents
 
-- [[[a19493,1]]]JSR-220:
-Enterprise JavaBeans, v. 3.0. Java Persistence API.
-_http://jcp.org/en/jsr/detail?id=220_.
-- [[[a19494,2]]]SQL 2003, Part 2,
-Foundation (SQL/Foundation). ISO/IEC 9075-2:2003.
-- [[[a19496,3]]]JDBC 4.2
-Specification. http://jcp.org/en/jsr/detail?id=221.
-- [[[a19497,4]]]Enterprise JavaBeans, v.
-2.1. _http://jcp.org/en/jsr/detail?id=153_.
-- [[[a19498,5]]]JSR-380: Bean Validation,
-v. 2.0. _http://jcp.org/en/jsr/detail?id=380_.
-- [[[a19499,6]]]JSR-366: Java Platform,
-Enterprise Edition 8 (Java EE 8) Specification.
-_http://jcp.org/en/jsr/detail?id=366[http://jcp.org/en/jsr/detail?id=366.]_
-- [[[a19500,7]]]JSR-365: Context and
-Dependency Injection for Java EE, v 2.0.
-_http://jcp.org/en/jsr/detail?id=365[http://jcp.org/en/jsr/detail?id=365.]_
-- [[[a19501,8]]]JSR-317: Java Persistence
-2.0. _http://jcp.org/en/jsr/detail?id=317_.
\ No newline at end of file
+- [[[a19493,1]]] Enterprise JavaBeans, v. 3.0. Java Persistence API.
+- [[[a19494,2]]] SQL 2003, Part 2, Foundation (SQL/Foundation). ISO/IEC 9075-2:2003.
+- [[[a19496,3]]] JDBC 4.2 Specification. http://jcp.org/en/jsr/detail?id=221.
+- [[[a19497,4]]] Enterprise JavaBeans, v. 2.1.
+- [[[a19498,5]]] Jakarta Bean Validation, v. 3.0. _https://projects.eclipse.org/projects/ee4j.bean-validation_.
```

Comment:
shouldn't the link point to sth like https://jakarta.ee/specifications/bean-validation/3.0/ instead? Or https://jakarta.ee/specifications/bean-validation/ ? Similarly for others

Reply:
Right. I updated the Bean validation and Jakarta EE addresses based on your suggestion. But I can't find such a page for CDI.

Reply:
Forget about my last comment. I found it.

Reply:
@lukasj Do you think this can be merged? or it needs more update from my side?